doc/index.rst

```diff
@@ -37,4 +41,11 @@ You can also use a normal Haskell development environment on your computer to wr
    :caption: Plutus Reference
    :maxdepth: 2
 
+References
+----------
+
+1. We develop our software in Haskell_. If you need to gain a better understanding of this powerful programming language, a good place to start is the http://haskell.org site.
```

Comment:
I don't *disagree* with these, but I guess I'm not sure how worth it is to *incrementally* improve these docs: most of these pages need to be "written properly", and if we do that we'll probably have to redo this too.

Reply:
I think references wouldn't change that much, though.

Reply:
There is a JIRA issue for "Playground references", that's what the PR is about